README.md

```diff
@@ -38,98 +80,48 @@ end
 
 ### Sinatra
 
-Add to config.ru
+Add the middleware to your config.ru
 
 ```ruby
 use HealthMonitorMiddleware, "your_application_name"
 ```
 
 ### Padrino
 
-Add to config/apps.rb
+Add the middleware to config/apps.rb
 
 ```ruby
 Padrino.use(HealthMonitorMiddleware, "your_application_name")
 ```
 
-## Configure your dependencies
+## Usage
+
+Put the following configurations some place where they are loaded on startup,
+like `config/initializer/health_monitor_initializer.rb`
 
-Put into some place which is loaded on startup, like
-`config/initializer/health_monitor_initializer.rb`
+### Simple monitors
+These can be used to monitor a database connection or anything that can be in two
+states.
 
+Example:
 ```ruby
-HealthMonitorMiddleware.add("simple", name: "MySQL") { User.where{id > 0}.limit(1) }
-
-HealthMonitorMiddleware.add("service", name: 'some_service_name') { RestClient.get(File.join(service-url, 'healthmonitor')) }
+HealthMonitorMiddleware.add("simple", name: "MySQL") do
+  ActiveRecord::Base.connected?
+end
 ```
 
-Simple-Monitor needs some block which returns true or false for up an down
-state. Service-Monitor needs an block which returns a complete status response
-in JSON format. This may be just the health-monitor used by a service.
-
-## Outcome
-
-As soon as the middleware is configured and your application is running, you can
-acess the health monitor status page adding 'health_monitor' to your url. It does
-not depend on the given path.
-
-    http://your_application_url_with_some_path/health_monitor
+### Service monitors
 
-This would lead to the following output:
+These can be used to monitor another service that is also using this gem. The block
+needs to return the result of the /healthmonitor endpoint of the other service:
```

Comment:
By default, this gem mounts itself to `/health_monitor`.  Please update this + the example according to that, and wrap the path in backticks